op-node/p2p/gossip_test.go

```diff
@@ -101,6 +102,59 @@ func TestVerifyBlockSignature(t *testing.T) {
 	})
 }
 
+func TestVerifyBlockSignatureWithRemoteSigner(t *testing.T) {
+	t.Skipf("Implement mocking of client rpc calls")
+
+	logger := testlog.Logger(t, log.LevelCrit)
+	cfg := &rollup.Config{
+		L2ChainID: big.NewInt(100),
+	}
+	peerId := peer.ID("foo")
+	secrets, err := e2eutils.DefaultMnemonicConfig.Secrets()
+	require.NoError(t, err)
+	msg := []byte("any msg")
+
+	t.Run("Valid", func(t *testing.T) {
+		runCfg := &testutils.MockRuntimeConfig{P2PSeqAddress: crypto.PubkeyToAddress(secrets.SequencerP2P.PublicKey)}
+		remoteSigner, err := NewRemoteSigner(logger, opsigner.NewCLIConfig())
+		require.NoError(t, err)
+		signer := &PreparedSigner{Signer: remoteSigner}
+		sig, err := signer.Sign(context.Background(), SigningDomainBlocksV1, cfg.L2ChainID, msg)
+		require.NoError(t, err)
+		result := verifyBlockSignature(logger, cfg, runCfg, peerId, sig[:65], msg)
```

Comment:
Why `[:65]` and not just `[:]` ?

Reply:
you're right, modified it to be just `[:]` 👍